engine/runtime/src/main/java/org/enso/interpreter/instrument/HostObjectDebugWrapper.java

```diff
@@ -0,0 +1,104 @@
+package org.enso.interpreter.instrument;
+
+import com.oracle.truffle.api.CompilerDirectives.TruffleBoundary;
+import com.oracle.truffle.api.TruffleLanguage;
+import com.oracle.truffle.api.TruffleLanguage.Env;
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.TruffleObject;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import com.oracle.truffle.api.library.ExportLibrary;
+import com.oracle.truffle.api.library.ExportMessage;
+import java.util.Arrays;
+import org.enso.interpreter.EnsoLanguage;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+
+/**
+ * Wrapper for host objects. Is a workaround for a bug in chromeinspector
+ * (https://github.com/oracle/graal/issues/5513). All the host objects should
```

Comment:
We should push towards fixing the issue in GraalVM. I'd like the fix to be in 23.0 version.